src/utils/appUtils.ts

```diff
@@ -7,8 +7,10 @@ import type { RCPage } from "../types/pageTypes"
  * conditionally rendering the Search.
  */
 export const getActivePage = (pathname: string): RCPage => {
-  if (pathname === "/" || pathname.includes("search")) {
+  if (pathname === "/" || pathname === "/search") {
```

Comment:
i like the specificity of === "/search" instead of includes("search"). Is there a reason the other pathnames are using includes instead of the === check?

Reply:
The includes matches substrings in the case of sub paths (e.g.  /404/redirect) but we can definitely use exact matchers in cases where there are none (e.g. "/search/advanced")

Reply:
I added a constant with the absolute paths, using those with exact matching which i think will be less flakey than .includes. We can add a regex later if we dealing with a bunch of sub paths but don't think we'll need to for this use case.